src/test/java/pages/ProfilePage.java

```diff
@@ -7,21 +7,27 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class ProfilePage extends BasePage{
-
-    //locators
-    By violetTheme = By.xpath("//div[@data-testid='theme-card-violet']");
-
-    By isVioletThemeSelected = By.xpath("//div[@data-testid='theme-card-violet' and contains(@class,'selected')]");
-
     public ProfilePage(WebDriver givenDriver) {
         super(givenDriver);
     }
+    //locators
+    By violetTheme = By.xpath("//div[@data-testid='theme-card-violet']");
 
+    By isVioletThemeSelected = By.xpath("//div[@data-testid='theme-card-violet' and contains(@class,'selected')]");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```